src/widgets/about-course/ui/about-course-grid/about-course-grid.module.scss

```diff
@@ -1,68 +1,54 @@
-.about-grid {
+.about-course-grid {
   display: grid;
   grid-template-areas:
     'a a b'
     'c d e';
```

Comment:
```suggestion
    'for-everyone for-everyone worldwide-mentors'
    'free-education certificate';
```

Reply:
The titles of the cards sometimes differ on different pages, I think it will be confusing